db/seeds.rb

```diff
@@ -13,7 +13,7 @@
 AdminUser.create(email: "admin@admin.com", password: "password")
 
 1.upto(4) do |i|
-  kudo = Kudo.create(title: Faker::Beer.style, content: Faker::Coffee.notes, giver_id: "#{i}", receiver_id: "#{i+1}")
+  kudo = Kudo.create(title: Faker::Beer.style, content: Faker::Coffee.notes, giver_id: "#{i}", receiver_id: "#{i+1}", company_value_id: "1")
```

Comment:
consider following approach:

`company_value: CompanyValue.all.sample`

It should be safer, than using fixed value as an id :)

Reply:
I tried to do the same with `rand(1)` but it didn't work so I entered a constant value :)

Reply:
the nit could improve the performance:

```ruby
company_values= CompanyValue.all
comapny_values.sample
```